tasks/size_contracts.js

```diff
@@ -13,6 +13,23 @@ const formatSize = function (size) {
   return (size / 1024).toFixed(3);
 };
 
+const ansiRegex = function ({ onlyFirst = false } = {}) {
+  const pattern = [
+    '[\\u001B\\u009B][[\\]()#;?]*(?:(?:(?:(?:;[-a-zA-Z\\d\\/#&.:=?%@~_]+)*|[a-zA-Z\\d]+(?:;[-a-zA-Z\\d\\/#&.:=?%@~_]*)*)?\\u0007)',
+    '(?:(?:\\d{1,4}(?:;\\d{0,4})*)?[\\dA-PR-TZcf-nq-uy=><~]))'
+  ].join('|');
```

Comment:
Source for this regex?

Reply:
Source: [ansi-regex](https://github.com/chalk/ansi-regex/blob/main/index.js#L1-L8)

Reply:
Thanks, see comment above.